pipeline/terraform/test-environment/variables.tf

```
@@ -0,0 +1,8 @@
+variable "region" {
+  default = "eu-west-2"
+}
+
+variable "cluster_id" {}
+variable "task_execution_role" {}
+variable "build_id" {}
+variable "ecs_address" {}
```

I think this should be:
```suggestion
variable "ecr_address" {}
```

---

Updated